app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -44,47 +31,8 @@ const Notes = () => {
 
   return (
     <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add New Note"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
-          />
-          <Table
-            fetchNotes={fetchNotes}
-            notes={notes}
-            setSelectedNoteIds={setSelectedNoteIds}
-          />
-        </>
-      ) : (
-        <EmptyState
-          image={EmptyNotesListImage}
-          primaryAction={() => setShowNewNotePane(true)}
-          primaryActionLabel="Add New Note"
-          subtitle="Add your notes to send customized emails to them."
-          title="Looks like you don't have any notes!"
-        />
-      )}
+      <Header setShowNewNotePane={setShowNewNotePane} />
+      <Note NOTES_DATA={notes} />
```

Comment:
This is not right. Here you have named the component as `Note`, singular. Which means it takes care of a single note. But you have passed in plural, `notes`. It doesn't make any sense. Refactor.

Reply:
Made the changes.